public/javascripts/cohortVizController.js

```diff
@@ -38,6 +40,16 @@ var cohortVizController = function($scope, $http, $state) {
 
     var starsList = [];
     $scope.starData = formatPieData($scope.stars, starsList);
+
+    var engagePeerList = [];
+    $scope.engagePeerData = formatPieData($scope.engagePeer, engagePeerList);
+
+    var engageAdultList = [];
+    $scope.engageAdultData = formatPieData($scope.engageAdult, engageAdultList);
+
+    var engageContentList = [];
+    $scope.engageContentData = formatPieData($scope.engageContent, engageContentList);
+   
```

Comment:
Yeah that is actually all done in the next pull request :)  I wasn't sure it it belonged here or there, so sorry about that